modules/op-release-notes-1-13.adoc

```diff
@@ -0,0 +1,142 @@
+
+[id="pac-new-features-1-13_{context}"]
+=== {pac}
+
+* With this update, in {pac}, when using a CEL expression for advanced event matching (pipelinesascode.tekton.dev/on-cel-expression), you can use the `header` and `body` fields to access the full payload that is passed by the Git repository provider body. You can use this feature to filter events by any information that the Git repository sends.
++
+--
+:FeatureName: Using the header and body of the payload in CEL expressions for event matching
+include::snippets/technology-preview.adoc[]
+--
+
+* With this update, when a {pac} pipeline run is triggered by a push event, you can use `/test`, `/test branch:<branchname>`, `/retest`, `/retest branch:<branchname>`, `/cancel`, and `/cancel branch:<branchname>` commands on the corresponding commit comment to re-run or cancel the pipeline run.
+
+* With this update, when using {pac}, you can use remote tasks on remote pipelines. Therefore, you can reuse a complete remote pipeline across multiple repositories. You can override tasks from the remote pipeline by adding a task with the same name.
++
+--
+:FeatureName: Using remote tasks on remote pipelines and overriding tasks
+include::snippets/technology-preview.adoc[]
+--
+
+* With this update, when using {pac}, you can view improved information about CI pipeline runs in the Git repository provider. This information includes the namespace and the associated pipeline run.
+
+[id="breaking-changes-1-13_{context}"]
+== Breaking changes
+
+* Before this update, in {pac}, when using a policy group, users who were not a part of policy groups and not allowed explicitly but allowed to run the CI  (via org ownership or otherwise) could sometimes execute pipeline runs by creating events such as pull requests or by entering commands such as `ok_to_test`. With this update, if policy groups are configured, only users that are added to the required policy groups can execute pipeline runs, and users that are a part of the owner organization but not configured in policy groups cannot execute pipeline runs.
+
+[id="known-issues-1-13_{context}"]
+== Known issues
+
+* To enable keeping pods when a task run is cancelled, along with setting the `pipeline.options.configMaps.feature-flags.data.keep-pod-on-cancel` spec to `true` in the `TektonConfig` CR, you also need to set the `pipeline.enable-api-fields` spec to `alpha` in the `TektonConfig` CR.
+
+* If you enable keeping pods when a task run is cancelled, when a task run is cancelled because of a default timeout or because you set a timeput in the pipeline specification, {pipelines-shortname} deletes the pod.
+
+[id="fixed-issues-1-13_{context}"]
+== Fixed issues
+
+* Before this update, some secrets that pipeline runs use for Git authentication could be deleted from the cluster during a cleanup. With this update, a secret is deleted only when all pipeline runs that use it are deleted.
```

Comment:
Should there be another word between between "runs" and "use?" (In the first sentence as compared to what's in the second sentence.)

Reply:
"some secrets that pipeline runs use for Git authentication" is correct, because pipeline runs use secrets for Git authentication.

Reply:
It still reads uncomfortably to me, but that's probably me. I'll go with your knowledge in this area.